ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/dhcp_servers.schema.yml

```diff
@@ -23,6 +23,23 @@ keys:
           type: str
         dns_domain_name_ipv6:
           type: str
+        dns_server:
+          type: dict
+          keys:
+            ipv4_servers:
+              type: list
+              description: List of IPv4 DNS servers
+              items:
+                type: str
+                required: true
+                description: 'IPv4 address of DNS server'
+            ipv6_servers:
+              type: list
+              description: List of IPv6 dns servers
+              items:
+                type: str
+                required: true
+                description: 'IPv6 address of DNS server'
```

Comment:
Maybe it would be more consistent with the domain names etc (and simpler to input) to just create the two lists without nesting in a dict.
`dns_servers_ipv4` and `dns_servers_ipv6`

Reply:
And same consideration for the `tftp_server_files_ipv4`, `tftp_server_files_ipv6`.